packages/react-dom/src/events/__tests__/DOMPluginEventSystem-test.internal.js

```diff
@@ -2424,6 +2424,60 @@ describe('DOMPluginEventSystem', () => {
             expect(log).toEqual(['beforeblur', 'afterblur']);
           });
 
+          // @gate experimental
+          it('beforeblur has the correct propagation mechancis after a nested focused element is unmounted', () => {
```

Comment:
"correct" is vague, can you title this in a more specific way? Ideally a person unfamiliar with desirable behavior should be able to tell from a test what it is the test is supposed to verify. E.g. if a test had a bug, they should be able to figure out how to fix it.

I think maybe it's "beforeblur should skip handlers from a deleted subtree after the focused element is unmounted"?